static/tag/navigation.tag

```diff
@@ -114,7 +114,6 @@
         <workspace-table if={isScrennToShow('myWorkspaces')></workspace-table>
         <workspace-share-table if={isScrennToShow('sharedWorkspaces')></workspace-share-table>
         <workspace-editor if={isScrennToShow('workspace')}></workspace-editor>
-        <graph if={isScrennToShow('graph')}></graph>
```

Comment:
Impossible to access and when accessing through url directly, throw an error because not in a workspace.